basics/next-app-router/instrumentation-client.ts

```diff
@@ -9,4 +9,7 @@ posthog.init(process.env.NEXT_PUBLIC_POSTHOG_KEY!, {
   capture_exceptions: true,
   // Turn on debug in development mode
   debug: process.env.NODE_ENV === "development",
+  // Disable request batching in test environment
+  request_batching: false,
```

Comment:
This is really suspect. I plan on updating this to detect if we're in CI and disable batching + allow bot agents with environment variables.